subprojects/core/src/main/java/org/gradle/api/internal/file/copy/DefaultCopySpec.java

```diff
@@ -585,7 +588,7 @@ public FileTree getAllSource() {
 
         @Override
         public DuplicatesStrategy getDuplicatesStrategy() {
-            if (duplicatesStrategy != DuplicatesStrategy.INHERIT) {
+            if (duplicatesStrategy != DEFAULT_DUPLICATION_STRATEGY) {
```

Comment:
extracted to reflect the intent: if it's the default, we should ignore it here (might as well be `null` instead, but didn't want to change it, don't know about the implications)